src/MultiAgentReinforcementLearning/src/policies/multi_agent.jl

```diff
@@ -0,0 +1,53 @@
+export MultiAgentManager
+
+struct MultiAgentManager <: AbstractPolicy
+    agents::Dict{Any,Any}
```

Comment:
I never understood why in the original version, dict values can be type any…shouldn’t they be <: AbstractPolicy?

Reply:
Changes in commit [606c3ba](https://github.com/JuliaReinforcementLearning/ReinforcementLearning.jl/pull/834/commits/606c3baf4a539ec99ae2e6dec3dbd1ccb32af584)